docs/data-sources/role_metadata.md

```diff
@@ -0,0 +1,50 @@
+---
+page_title: "Mackerel: mackerel_role_metadata"
+subcategory: "Role"
+description: |-
+---
+
+# Data Source: mackerel_role_metadata
+
+Use this data source allows access to details of a specific Role Metadata.  
+
+## Example Usage
+
+```terraform
+resource "mackerel_service" "foo" {
+  name = "foo"
+}
+
+resource "mackerel_role" "bar" {
+  service = mackerel_service.foo.name
+  name = "bar"
```

Comment:
```suggestion
  service = mackerel_service.foo.name
  name    = "bar"
```

I think it's better to format.

Reply:
fix format https://github.com/mackerelio-labs/terraform-provider-mackerel/pull/50/commits/8c995d55d036079477c43da5982868e4878d13fe